.appveyor.yml

```diff
@@ -0,0 +1,35 @@
+environment:
+  matrix:
+    - nodejs_version: Current
+      platform: x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  - set CI=true
+  - npm install --global npm@latest
+  - set PATH=%APPDATA%\npm;%PATH%
+  - npm install
+  - npm install -g gulp istanbul codeclimate-test-reporter
+  - npm install ogg-packet
+
+matrix:
+  fast_finish: true
+
+platform:
+  - x64
```

Comment:
What's with the double x64 platform specification? It's defined here and under the environment as well.

I can't remember the Appveyor yml syntax, but if I wanted to add x86 platform, would I add it here or in the matrix? Why both?

Reply:
good point. I'll remove that one.